README.md

```diff
@@ -153,22 +152,33 @@ service Echo {
 }
 ```
 
-You can use it like so (enable async in the interactive shell first):
-
+A client can be implemented as follows:
 ```python
->>> import echo
->>> from grpclib.client import Channel
+import echo
+from grpclib.client import Channel
+import asyncio
 
->>> channel = Channel(host="127.0.0.1", port=1234)
->>> service = echo.EchoStub(channel)
->>> await service.echo(value="hello", extra_times=1)
-EchoResponse(values=["hello", "hello"])
 
->>> async for response in service.echo_stream(value="hello", extra_times=1)
+async def main():
+    channel = Channel(host="127.0.0.1", port=50051)
+    service = echo.EchoStub(channel)
+    print(await service.echo(value="hello", extra_times=1))
+
+    async for response in service.echo_stream(value="hello", extra_times=1):
         print(response)
 
-EchoStreamResponse(value="hello")
-EchoStreamResponse(value="hello")
+    # don't forget to close the channel when done!
+    channel.close()
+loop = asyncio.get_event_loop()
+loop.run_until_complete(main())
+
```

Comment:
```suggestion


if __name__ == '__main__':
    asyncio.run(main())

```

Reply:
i specifically didn't use `asyncio.run` (cause of second commit) as It is introduced into the standard library as of [python 3.7](https://docs.python.org/3/library/asyncio-task.html#asyncio.run), since this library targets >= 3.6 this method cannot be used without breaking in that older interpreter. 

Moving the runtime into a entry point check, however, is a good idea.